.gitattributes

```diff
@@ -0,0 +1,19 @@
+* text=auto
+
+/.vscode export-ignore
+/fixtures export-ignore
+/tests export-ignore
+/images export-ignore
+/validation export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.gitmodules export-ignore
+/.dockerignore export-ignore
+/.travis.yml export-ignore
+/codecov.yml export-ignore
+/Dockerfile export-ignore
+/Performance.php export-ignore
+/php.ini export-ignore
+/phpcs.xml.dist export-ignore
+/phpunit.xml.dist export-ignore
+/README.md export-ignore
```

Comment:
Isn't a README always good to have in a distribution?

Reply:
It is common to exclude this file as well. Think of this as when the package is installed in production. When there is no need to check the `vendor` directory.

Reply:
True, but does this setting only apply to production environments?

Reply:
Yes, this will apply if the user installs packages with the `--prefer-dist` flag or config.

Reply:
gotcha, that makes sense to me.